VKClone/ViewControllers/VKAuthViewController.swift

```diff
@@ -29,15 +29,19 @@ final class VKAuthViewController: UIViewController {
     // MARK: - IBActions
 
     @IBAction private func loadFriendAction() {
-        apiService.fetchMyFriends()
+        apiService.fetchMyFriends { friends in
+            debugPrint(friends)
```

Comment:
```suggestion
            debugPrint(friends)
```
друзья не грузятся в таблицу, только печатаются?

Reply:
Тут да, в этом контроллере это из прошлого дз получается где надо было просто принтовать ответ, я выполнил это в отдельном контроллере AuthViewController, могу удалить его если более не понадобится